### PR TITLE
Added QRNG FAQ, `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by vanshwassan**
- Added FAQ as mentioned in [this issue](https://github.com/api3dao/vitepress-docs/issues/627#issue-1899887915). Tried to revamp the getting started section to include both FAQ and the [qrng-example](https://github.com/api3dao/qrng-example) repo within the same page.
- Also added a `CONTRIBUTING.md` markdown for contributors to refer to before making a contribution.

### CODE REVIEW

File: docs/guides/qrng/index.md [hunk 1]
```diff
@@ -65,16 +98,76 @@ additional functionality.
...
+QRNG Airnode, you can use the `endpointIdUint256` to return a single random
+number and have it modulo the range difference and add the minimum number of the
+range. Refer to the code snippet below to see how to convert the random number
+and have it within the range `[x, y]`.
+
+```solidity
+uint256 randomNumber = (qrngUint256 % (y - x + 1)) + x;
+// y is the max number for the range, x is min number for the range.
+// the randomNumber will be in range [x, y].
+```
```
Comment (dcroote) [inline docs/guides/qrng/index.md:171]: Nice

File: docs/guides/qrng/index.md [hunk 2]
```diff
@@ -65,16 +98,76 @@ additional functionality.
...
+| Fantom testnet            | 0.5    | FTM   | 4002     |
+| Avalanche Fuji testnet    | 0.3    | AVAX  | 43113    |
+| Polygon Mumbai testnet    | 0.05   | MATIC | 80001    |
+| Milkomeda C1 testnet      | 0.5    | mTAda | 200101   |
+| Arbitrum testnet          | 0.01   | AGOR  | 421613   |
+
+:::
+
+You can read more about `sponsorWallet`
+[here](/reference/airnode/latest/concepts/sponsor.html#sponsorwallet).
```
Comment (dcroote) [inline docs/guides/qrng/index.md:139]:
```suggestion
[here](/reference/airnode/latest/concepts/sponsor.md#sponsorwallet).
```

File: docs/guides/qrng/index.md [hunk 3]
```diff
@@ -65,16 +98,76 @@ additional functionality.
...
+and requesting a withdrawal from the Airnode.
+
+## How much does it cost to use QRNG?
+
+QRNG is free to use. You can use it as much as you want without paying anything.
+However, you will have to fund the `sponsorWallet` that incurs the gas fees for
+the transaction for the fulfillment of the request. A `sponsorWallet` address is
+derived from the requester contract address, the QRNG Airnode address and its
+extended public key (xpub). A sponsor wallet must be derived using the command
+[derive-sponsor-wallet-address](/reference/airnode/latest/packages/admin-cli.html#derive-sponsor-wallet-address)
```
Comment (dcroote) [inline docs/guides/qrng/index.md:113]:
```suggestion
[derive-sponsor-wallet-address](/reference/airnode/latest/packages/admin-cli.md#derive-sponsor-wallet-address)
```

**Review by dcroote — CHANGES_REQUESTED** [review]

**Review by dcroote — COMMENTED** [review]
Nice FAQ additions.  For `CONTRIBUTING.md`, the content overlaps some with https://docs.api3.org/dev/ - perhaps link to the relevant pages from the appropriate `CONTRIBUTING.md` sections?

**Comment by vanshwassan** [comment]
> Nice FAQ additions. For `CONTRIBUTING.md`, the content overlaps some with https://docs.api3.org/dev/ - perhaps link to the relevant pages from the appropriate `CONTRIBUTING.md` sections?

Ah I didn't know about this section, added references to this from `CONTRIBUTING.md`.

File: docs/guides/qrng/index.md [hunk 4]
```diff
@@ -16,43 +17,75 @@ tags:
...
-## Sponsor Wallet
+### Sponsor Wallet
 
 The QRNG example project
 [sets the sponsor wallet<ExternalLinkImage/>](https://github.com/api3dao/qrng-example/blob/main/deploy/2_setup.js#L11-L28)
 using the requester address. The wallet is then used to pay gas costs when the
 Airnode responds to a request. An alternate method is to use the
 [Admin CLI](/reference/airnode/latest/packages/admin-cli.md) as is the case with
-the [Using QRNG - Remix Example](/guides/qrng/qrng-remix/index.md) guide.
+the [Using QRNG - Remix Example](/guides/qrng/qrng-remix/index.md/) guide.
```
Comment (dcroote) [inline docs/guides/qrng/index.md:84]:
```suggestion
the [Using QRNG - Remix Example](/guides/qrng/qrng-remix/index.md) guide.
```

File: docs/guides/qrng/index.md [hunk 5]
```diff
@@ -16,43 +17,75 @@ tags:
...
-requests a quantum random number.
+This is the getting started section for QRNG along with some frequently asked
+questions. If you have a question that is not answered here, you can head over
+to our [Discord server](https://discord.com/invite/qnRrcfnm5W).
+
+## What is the QRNG?
+
+QRNG stands for Quantum Random Number Generator. It is a free to use service
+that provides quantum randomness on-chain. It is powered by
+[Airnode](/reference/airnode/latest/understand/index.md/), the
```
Comment (dcroote) [inline docs/guides/qrng/index.md:28]:
```suggestion
[Airnode](/reference/airnode/latest/understand/index.md), the
```

File: docs/guides/qrng/index.md [hunk 6]
```diff
@@ -16,43 +17,75 @@ tags:
...
+[first-party](/explore/airnode/why-first-party-oracles.md) oracle that is
+directly operated by the QRNG API providers. You can read more about QRNG
+[here](/reference/qrng/index.md/).
```
Comment (dcroote) [inline docs/guides/qrng/index.md:31]:
```suggestion
[here](/reference/qrng/index.md).
```

**Review by dcroote — APPROVED** [review]
A few links need the trailing `/` removed, otherwise nice 👍

**Comment by dcroote** [comment]
Any reason not to merge?

**Review by Ashar2shahid — APPROVED** [review]

**Comment by Ashar2shahid** [comment]
Looks good, can get it merged

**Pull request merged**